tests/datastore_mysql/tox.ini

```
@@ -0,0 +1,24 @@
+[tox]
+setupdir = {toxinidir}/../..
+envlist =
+    {py27,py35,py36,py37,py38,py39},
```

```suggestion
    {py27,py35,py36,py37,py38},
```

Package doesn't support 3.9, we can drop testing for it.